src/libc/src/stdio.c

```diff
@@ -31,10 +41,19 @@ size_t fwrite(const void *ptr, size_t size, size_t nmemb, FILE *stream) {
   return 0;
 }
 char *fgets(char *s, int size, FILE *stream) { return 0; }
+long ftell(FILE *stream) { return 0; }
 
 FILE *popen(const char *command, const char *type) { return 0; }
 int pclose(FILE *stream) { return 0; }
 
+int vfprintf(FILE *restrict stream, const char *restrict format, va_list ap) {
+  return 0;
+}
 int vsnprintf(char *str, size_t size, const char *format, va_list ap) {
   return 0;
 }
+
+FILE memstream = {0};
+FILE *open_memstream(char **ptr, size_t *sizeloc) {
+  return &memstream;
+}
```

Comment:
Originally I tried to write this as:

```c
FILE *open_memstream(char **ptr, size_t *sizeloc) {
  static FILE f = {0};
  return &f;
}
```

but this caused some runtime memory owi errors.

Reply:
cc @filipeom, this is where we got a "cannot fetch base pointer" error.